src/shogun/mathematics/linalg/LinalgBackendEigen.h

```diff
@@ -688,6 +704,22 @@ namespace shogun
 		template <typename T>
 		void log_impl(const SGMatrix<T>& a, SGMatrix<T>& result) const;
 
+		/** Eigen3 matrix sin method */
```

Comment:
maybe we dont need to say "Eigen3" here since the file is for eigen (I know you just took that from the other code, but still)

Reply:
Off the record, I can't say I understood our linalg to its entirety(which i'm planning to change :D),  so I did a quick swoop from ahmeds code for the time being. Anyway you want me to change all such occurrences or only the one you mentioned?

Reply:
no need to change this everywhere (in general when we make comments you dont need to change other code, only your own)